static/src/inline-svgs/icon/mail.svg

```diff
@@ -1 +1 @@
-<svg width="14" height="14" viewBox="-298 390 14 14"><path d="M-286.2 395.6l-.3-.3-4.4 2.1-4.4-2.1-.2.3 4.7 3.8c-.1 0 4.6-3.8 4.6-3.8zm1-3.7h-11.6l-1.2 1.9v7.2l1.2.9h11.6l1.2-.9v-7.2c0 .1-1.2-1.9-1.2-1.9zm-.4 8.4h-10.7v-6.7h10.7v6.7z"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="10" viewBox="4 3 14 10"><path d="M16.728 11.75H5.272V4.875l5.09 4.375h1.274l5.09-4.375v6.875zm-1.274-7.5L11 8 6.546 4.25h8.908zM16.728 3H5.272L4 4.25v7.5L5.272 13h11.455L18 11.75v-7.5L16.728 3z"/></svg>
```

Comment:
Have a look at this doc on svgs: https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md#image-preparation

I think the `xmlns` stuff needs to be removed.